kubeflow-pipelines/pipeline-code/ray-code.py

```diff
@@ -28,13 +28,68 @@ def initialize_ray():
         ray.util.disconnect()
 
     ray.util.connect('{ray_head}:10001'.format(ray_head='ray-cluster-kubeflow-ray-head'))
+
+
+    # Prerequisite - Specify reusable definitions
+
+    metricName = 'cluster_version'
+    #metricName = 'cluster_feature_set'
+    #metricName = 'cluster_infrastructure_provider'
+    #metricName = 'cluster:usage:consumption:rhods:cpu:seconds:rate5m'
+
+    bucketName = 'DH-SECURE-THANOS-RAY-USE'
+    endpoint = 'https://s3.upshift.redhat.com'
+
+    prefixPathRead = 'raydev'
+    prefixPathWrite = 'raydev-write-demo'
+
+    year = '2021'
+    month = '01'
+    day = '01'
+
+    # Read path
+    read_path = f'{prefixPathRead}/metric={metricName}'
+
+    # Write path
+    write_path = f'{prefixPathWrite}/metric={metricName}'
+
+    # Prerequisite - Create S3FileSystem in PyArrow
+    # Why: Allows us to specify a custom endpoint
+    AWS_ACCESS_KEY = "<Ceph S3 Creds Here>"
```

Comment:
could you use `os.environ['AWS_ACCESS_KEY']` here?

Reply:
The image doesn't come with it preloaded, so I would have to set it from somewhere after the pod is created. 
I have the secret up too, if only I could spec the pipeline python code to grab it when the pod is created.